pkg/addons/data/DaemonSet.apps/kube-service-catalog/apiserver.yaml

```diff
@@ -25,29 +26,48 @@ spec:
         - https://master-000001:2379
         - https://master-000002:2379
         - --etcd-cafile
-        - /var/run/kubernetes-service-catalog/etcd-ca.crt
+        - /etc/origin/master/master.etcd-ca.crt
         - --etcd-certfile
-        - /var/run/kubernetes-service-catalog/etcd-client.crt
+        - /etc/origin/master/master.etcd-client.crt
         - --etcd-keyfile
-        - /var/run/kubernetes-service-catalog/etcd-client.key
+        - /etc/origin/master/master.etcd-client.key
         - -v
         - "3"
+        - --cors-allowed-origins
+        - localhost
         - --enable-admission-plugins
-        - KubernetesNamespaceLifecycle,DefaultServicePlan,ServiceBindingsLifecycle,ServicePlanChangeValidator,BrokerAuthSarCheck
+        - NamespaceLifecycle,DefaultServicePlan,ServiceBindingsLifecycle,ServicePlanChangeValidator,BrokerAuthSarCheck
         - --feature-gates
         - OriginatingIdentity=true
+        - --feature-gates
+        - NamespacedServiceBroker=true
         command:
         - /usr/bin/service-catalog
         image: '*** GENERATED ***'
         name: apiserver
+        ports:
+        - containerPort: 6443
         volumeMounts:
         - mountPath: /var/run/kubernetes-service-catalog
           name: apiserver-ssl
           readOnly: true
+        - mountPath: /etc/origin/master
+          name: etcd-host-cert
+          readOnly: true
       nodeSelector:
-        node-role.kubernetes.io/infra: "true"
+        node-role.kubernetes.io/master: "true"
```

Comment:
The biggest problem with running things in master is that etcd is very sensitive in high load so we probably want to avoid running anything more than the core control plane.

Reply:
validate if needed to be co-located with etcd. validate how it was in 3.10. 
Lets stick with upstream.

Reply:
in 3.11 it is not hosted on "master" so we would not need to keep etcd certs in the secret. Agreed to keep it this way so we do the same.